packages/strapi-admin/admin/src/translations/ja.json

```diff
@@ -203,6 +204,7 @@
   "notification.error.layout": "レイアウトを復旧できませんでした",
   "notification.form.error.fields": "フォームに同じエラーがあります",
   "notification.form.success.fields": "保存されました",
+  "notification.link-copied": "クリップボードにリンクをコピーしました",
```

Comment:
According to google translate... This traduction says 'I copied the link..' Can you remove all first person from your translations?

Reply:
@soupette 

Thank you for the review.

This sentence does not include the wording of the first person.
This is a common Japanese phrase.

<img src=https://user-images.githubusercontent.com/24698894/88663764-0c5b8980-d117-11ea-8994-116918f6ed93.png  width=400>

Other sentences are the same.

Reply:
Ok I understand!